.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,16 @@
+Thanks for contributing code! Please include a description of your change and
+check your PR against the list below (for further questions, refer to the
+[contributing guide](CONTRIBUTING.md)).
```

Comment:
Make sure fix this link to point to the right location. Maybe an absolute link would be better (https://...)

Reply:
Now that the files are back to their original location, this would be `[contributing guide](../CONTRIBUTING.md)`, but I'm fine with the absolute links as suggested. At this point, the links are absolute.

Reply:
PS: while I think the absolute link is not as elegant as the relative one, the advantage of the former is that it will always point to the 'official' versions.